nifi-nar-bundles/nifi-soap-bundle/nifi-soap-processors/src/main/java/org/apache/nifi/processors/soap/GetSOAP.java

```diff
@@ -46,34 +51,30 @@
 import org.apache.nifi.processor.ProcessorInitializationContext;
 import org.apache.nifi.processor.Relationship;
 import org.apache.nifi.processor.exception.ProcessException;
-import org.apache.nifi.processor.io.OutputStreamCallback;
 import org.apache.nifi.processor.util.StandardValidators;
-import org.apache.nifi.util.StopWatch;
+
 import java.io.IOException;
-import java.io.OutputStream;
 import java.util.*;
 
-import static org.apache.axis2.namespace.Constants.URI_SOAP12_ENV;
-
-
 @SupportsBatching
-@InputRequirement(InputRequirement.Requirement.INPUT_FORBIDDEN)
+@InputRequirement(InputRequirement.Requirement.INPUT_REQUIRED)
 @Tags({"SOAP", "Get", "Ingest", "Ingress"})
-@CapabilityDescription("Execute provided request against the SOAP endpoint. The result will be left in it's orginal form. " +
+@CapabilityDescription(
+        "Execute provided request against the SOAP endpoint. The result will be left in it's orginal form. " +
```

Comment:
Do we want to indicated that any headers coming back will be written as attributes to the outgoing FF? Not sure if we need to mention how it can be triggered as that is handled by the UI based on annotations, also the scheduling may change and thus render this documentation obsolete.